packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -44,11 +44,12 @@
   },
   "callWithChat": {
     "peopleButtonLabel": "People",
-    "peopleButtonTooltipContentOpen": "Show Participants",
-    "peopleButtonTooltipContentClose": "Hide Participants",
+    "peopleButtonTooltipOpen": "Show participants",
+    "peopleButtonTooltipClose": "Hide participants",
     "chatButtonLabel": "Chat",
-    "chatButtonTooltipContentOpen": "Show Chat",
-    "chatButtonTooltipContentClose": "Hide Chat",
+    "chatButtonNewMessageCountToolTip": "Show chat ({unreadMessagesCount} unread)",
```

Comment:
suggestion: This string is actually related to `chatButtonTooltipOpen` (it adds stuff on top).
A name like `chatButtonTooltipOpenWithMessageCount` would better reflect that than `chatButtonNewMessageCountTooltip`.

Reply:
kk will update the name! names are hard 😊